dev/n_osync.sh

```diff
@@ -287,6 +287,9 @@ function _CheckReplicasLocal {
 	if [ $MINIMUM_SPACE -ne 0 ]; then
 		Logger "Checking minimum disk space in local replica [$replicaPath]." "NOTICE"
 		diskSpace=$($DF_CMD "$replicaPath" | tail -1 | awk '{print $4}')
+		if [[ $diskSpace == *"%"* ]]; then
+	  		diskSpace=$($DF_CMD "$replicaPath" | tail -1 | awk '{print $3}')
+		fi
```

Comment:
Can you provide a `df /var` output from a QNAP to make sure what we're talking about ?

Reply:
For Qnap, the output for the NAS shared folders does not include a mount point (honestly i have no idea how it works under the hoods) this points to the disk containing the /share/Public "virtual disk"
[admin@GELE ~]# df "/share/Public/Edr/PortableApps" | tail -1
                          5.8T      3.3T      2.5T  57% /share/CACHEDEV1_DATA
 
When asking instead the df on regular mount points/folders everything works, but usually the targets are the former        
[admin@GELE ~]# df
Filesystem                Size      Used Available Use% Mounted on
none                    400.0M    275.8M    124.2M  69% /
devtmpfs                  7.7G      8.0K      7.7G   0% /dev
....

On cygwin instead, and on most systems everything works flawlessy
YYI9912@2020-150754 ~
$ df "/cygdrive/c/Private/PortableApps" | tail -1
C:             246914044 197654740  49259304  81% /cygdrive/c

May be taking the penultimate parameter in awk would work...should find how. The "%" sign was the easier solution

Reply:
Thanks for the input, looks fair enough